internal/mode/static/telemetry/collector_test.go

```diff
@@ -239,6 +253,18 @@ var _ = Describe("Collector", Ordered, func() {
 		})
 	})
 
+	Describe("clusterID collector", func() {
+		When("collecting clusterID", func() {
+			It("throws an error when collecting clusterID", func() {
```

Comment:
nit: just to keep consistency with the other tests

Can you add another `When("it encounters an error while collecting data")` which wraps all the error tests? (can look below for examples)

```suggestion
			It("should error if the kubernetes client errored when getting the namespace", func() {
```

Reply:
This hasn't been committed

Reply:
This hasn't been addressed

Reply:
weird, i committed this twice. I'll change it locally and push